integration-tests/mockbean/src/test/java/io/quarkus/it/mockbean/GreetingResourceTest.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.it.mockbean;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.MockBean;
+
+@QuarkusTest
+class GreetingResourceTest {
+
+    @MockBean
+    MessageService messageService;
```

Comment:
Currently, you can do:
```java
@Produces
@Mock
MessageService mockMessageService() {
   return Mockito.mock(MessageService.class);
}

@Inject
Message
```

Reply:
Yes absolutely!

The PR doesn't bring in fundamental capability that didn't already exist. It does however lighten the burden for users if they have multiple mocks they want to use in their tests (whether that is good or bad is another question :) )